app/test_split_integer.py

```diff
@@ -2,20 +2,21 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(22, 4)) == 22
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert 8 == max(split_integer(8, 4)) * 4
```

Comment:
`split_integer` should be placed left-side. Besides that, compare directly with a list, do not perform additional calculations